pytorch/inference/docker/2.2/py3/Dockerfile.graviton.cpu

```diff
@@ -0,0 +1,253 @@
+ARG PYTHON=python3
+ARG PYTHON_VERSION=3.10.9
+ARG MAMBA_VERSION=22.11.1-4
+ARG OPEN_MPI_VERSION=4.1.5
+
+# PyTorch Graviton Binaries
+ARG TORCH_URL=https://framework-binaries.s3.us-west-2.amazonaws.com/pytorch/v2.2.1/arm64_cpu/torch-2.2.1%2Bcpu-cp310-cp310-manylinux2014_aarch64.whl
+ARG TORCHVISION_URL=https://framework-binaries.s3.us-west-2.amazonaws.com/pytorch/v2.2.1/arm64_cpu/torchvision-0.17.1%2Bcpu-cp310-cp310-manylinux2014_aarch64.whl
+ARG TORCHAUDIO_URL=https://framework-binaries.s3.us-west-2.amazonaws.com/pytorch/v2.2.1/arm64_cpu/torchaudio-2.2.1%2Bcpu-cp310-cp310-manylinux2014_aarch64.whl
+ARG TORCHTEXT_URL=https://framework-binaries.s3.us-west-2.amazonaws.com/pytorch/v2.2.1/arm64_cpu/torchtext-0.17.1%2Bcpu-cp310-cp310-manylinux2014_aarch64.whl
+ARG TORCHSERVE_VERSION
+ARG SM_TOOLKIT_VERSION
+
+########################################################
+#  _____ ____ ____    ___
+# | ____/ ___|___ \  |_ _|_ __ ___   __ _  __ _  ___
+# |  _|| |     __) |  | || '_ ` _ \ / _` |/ _` |/ _ \
+# | |__| |___ / __/   | || | | | | | (_| | (_| |  __/
+# |_____\____|_____| |___|_| |_| |_|\__,_|\__, |\___|
+#                                         |___/
+#  ____           _
+# |  _ \ ___  ___(_)_ __   ___ 
+# | |_) / _ \/ __| | '_ \ / _ \
+# |  _ <  __/ (__| | |_) |  __/
+# |_| \_\___|\___|_| .__/ \___|
+#                  |_|
+########################################################
+FROM arm64v8/ubuntu:20.04 as ec2
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG PYTHON
+ARG PYTHON_VERSION
+ARG OPEN_MPI_VERSION
+ARG MAMBA_VERSION
+
+# PyTorch Passed arguments (URLs and Versions)
+ARG TORCH_URL
+ARG TORCHVISION_URL
+ARG TORCHAUDIO_URL
+ARG TORCHTEXT_URL
+ARG TORCHSERVE_VERSION
+
+### Common ENV Variables
+ENV DEBIAN_FRONTEND=noninteractive \
+    LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib" \
+    LANG=C.UTF-8 \
+    LD_LIBRARY_PATH=/opt/conda/lib/:$LD_LIBRARY_PATH \
+    PATH=/opt/conda/bin:$PATH \
+    TEMP=/home/model-server/tmp \
```

Comment:
TS 0.9.0 config update.

Just commenting as a bookmark so we don't forget but no change for now until we hear from @lxning about `TS>=0.9.0` config update.